src/parsers/EN/ENDeadlineFormatParser.js

```diff
@@ -7,11 +7,11 @@ var moment = require('moment');
 var Parser = require('../parser').Parser;
 var ParsedResult = require('../../result').ParsedResult;
 
-var PATTERN = /(\W|^)(within|in)\s*([0-9]+|an?)\s*(minutes?|hours?|days?)\s*(?=(?:\W|$))/i;
+var PATTERN = /(\W|^)(within|in)\s*([0-9]+|an?|half\s*an?|half)\s*(minutes?|hours?|days?)\s*(?=(?:\W|$))/i;
```

Comment:
Will `([0-9]+|an?|half(?:\s*an?)?)` works? I prefer not to have "half" twice in regex.